nullaway/src/main/java/com/uber/nullaway/fixserialization/out/SuggestedNullableFixInfo.java

```diff
@@ -25,23 +25,23 @@
 import com.google.errorprone.util.ASTHelpers;
 import com.sun.source.util.TreePath;
 import com.uber.nullaway.ErrorMessage;
-import com.uber.nullaway.fixserialization.location.FixLocation;
+import com.uber.nullaway.fixserialization.location.SymbolLocation;
 import java.util.Objects;
 
 /** Stores information suggesting adding @Nullable on an element in source code. */
 public class SuggestedNullableFixInfo {
 
   /** FixLocation of the target element in source code. */
```

Comment:
This comment is outdated given the renaming.

Reply:
ddb2afa048fc46566c3b0038bc714e7f63d51a5d

Also ran `git grep FixLocation`, there is no usage of `FixLocation` anymore.